SAM7s_base/usart_at91/usart_device_at91.c

```diff
@@ -142,9 +134,17 @@ static unsigned int createUartMode(unsigned int bits, unsigned int parity, unsig
 int usart_device_init()
 {
     AT91F_PMC_EnablePeriphClock( AT91C_BASE_PMC,(1 << AT91C_ID_US0) | (1 << AT91C_ID_US1));
-    if (!initQueues()) return 0;
+
+    /* Init device 0 (Telemetry) */
+    if (!init_txrx_queue(0, 512, 512))
```

Comment:
It's a best practice for readability and maintainability. http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad